src/js/components/AppListComponent.jsx

```diff
@@ -168,7 +170,9 @@ var AppListComponent = React.createClass({
   },
 
   hasFilters: function () {
-    return Object.values(this.props.filters).some(filter => {
+    var filters = this.getQueryParamObject();
+
+    return Object.values(filters).some(filter => {
```

Comment:
Could you add an extra check to ensure we are testing on a filter, because in the filters-object, that is actually the query params object now, could be every query param.

```JS
Object.values(FilterTypes).some(filterType => filterType === filter) &&
```